collector/src/test/java/io/prometheus/jmx/ExistDbMXBean.java

```diff
@@ -0,0 +1,109 @@
+package io.prometheus.jmx;
+
+import javax.management.MBeanServer;
+import javax.management.ObjectName;
+import java.util.Map;
+import java.util.TreeMap;
+
+public interface ExistDbMXBean {
+    public Map<QueryKey, RunningQuery> getRunningQueries();
```

Comment:
Add line. `public` is not required for an interface method.

Reply:
@dhoard I kept these as your other MBean interfaces have them. I thought these might be needed for your `java6` compatibility?

Reply:
Ah I see, you removed `java6` compatibility in the `development` branch.

Reply:
Done.